src/Fantomas.Core/CodePrinter.fs

```diff
@@ -3263,61 +3259,65 @@ let genTypeDefn (td: TypeDefn) =
             (indentSepNlnUnindent (sepNlnTypeAndMembers typeDefnNode +> genMemberDefnList members))
         |> genNode node
     | TypeDefn.Record node ->
-        let smallExpression =
-            sepSpace
-            +> genAccessOpt node.Accessibility
-            +> sepSpace
-            +> genSingleTextNode node.OpeningBrace
-            +> addSpaceIfSpaceAroundDelimiter
-            +> col sepSemi node.Fields genField
-            +> addSpaceIfSpaceAroundDelimiter
-            +> genSingleTextNode node.ClosingBrace
-
         let multilineExpression (ctx: Context) =
+            let noMembers = List.isEmpty members
```

Comment:
You shouldn't make any changes whatsoever to `TypeDefn.Record` in this PR.

Reply:
Sorry, leftover from trying to find the source of the issue 😅. Will change back